.gitignore

```diff
@@ -21,5 +21,6 @@ bin
 *.swp
 *.swo
 *~
+.vscode
```

Comment:
It's possible to ignore folders system wide rather than per repository: https://stackoverflow.com/a/7335487
Then it's not needed on every repository. ☺️

Reply:
Thank you for the tip 😊